docs/en/observability/synthetics-role-read.asciidoc

```diff
@@ -39,7 +39,10 @@ and grant the following privileges:
 | Read-only access to synthetics alert indices.
 
 | {kibana-ref}/kibana-privileges.html[Kibana]
-| `Uptime/Synthetics`: `All`
+| `Synthetics and Uptime`: `All`
 | Access to the {synthetics-app} in {kib}.
 
 |====
+
+// What does it mean for "Elastic managed locations enabled" to be unchecked for the read role?
+// Can they not _view_ monitors hosted on Elastic's global managed infrastructure if they exist?
```

Comment:
What does it mean for "Elastic managed locations enabled" to be unchecked for the [_read_ role](https://observability-docs_3305.docs-preview.app.elstc.co/guide/en/observability/master/synthetics-role-read.html)? Can they not _view_ monitors hosted on Elastic's global managed infrastructure if they exist?

Reply:
i think as far as under `read` i don't think it will have any impact on user experience. We don't hide any monitors from the users if they have permissions. Only adding/editing/deletion part is impacted.
Because checkbox is related to locations not necessarily monitors. 

Ideally it should not be visible in read section.  i will check to see if that's possible.